rollup.config.js

```diff
@@ -4,49 +4,57 @@ import resolve from 'rollup-plugin-node-resolve';
 import commonjs from 'rollup-plugin-commonjs';
 import pkg from './package.json';
 
-const config = {
-	input: 'src/index.js',
-	plugins: [
-		json(),
-		commonjs(),
-		buble({
-			target: {
-				node: 4
-			},
-			include: [
-				'src/**',
-				'node_modules/acorn-object-spread/**',
-				'node_modules/unicode-loose-match/**',
-				'node_modules/regexpu-core/**'
-			],
-			transforms: {
-				dangerousForOf: true
-			}
-		}),
-		resolve()
-	],
-	name: 'buble',
-	sourcemap: true
+const createConfig = ({
+	browser = false,
+	external = ['acorn', 'magic-string'],
+	output,
+} = {}) => {
+	return {
+		input: 'src/index.js',
+		output,
+		external,
+		plugins: [
+			json(),
+			commonjs(),
+			buble({
+				target: !browser ? { node: 4 } : null,
+				include: [
+					'src/**',
+					'node_modules/acorn-object-spread/**',
+					'node_modules/unicode-loose-match/**',
+					'node_modules/regexpu-core/**'
+				],
+				transforms: {
+					dangerousForOf: true
+				}
+			}),
+			resolve()
+		],
+		name: 'buble',
+		sourcemap: true
+	};
 };
 
 export default [
-	/* ESM/UMD builds */
-	Object.assign({}, config, {
-		external: ['acorn/dist/acorn.js', 'magic-string'],
+	/* node ESM/CJS builds */
+	createConfig({
 		output: [
 			{ format: 'es', file: pkg.module },
-			{ format: 'umd', file: pkg.main }
+			{ format: 'cjs', file: pkg.main }
 		],
-		globals: {
```

Comment:
removed those, as `unpkg`/umd bundle do not use any externals and `globals` have only a meaning for umd/iife